scripts/scss-typography-generator.js

```diff
@@ -22,53 +29,48 @@ const getShortSize = (size) => {
 	if (size === 'xsmall') {
 		return 'xs';
 	}
-	if (size === 'xxsmall') {
+	if (size === '2xsmall') {
 		return '2xs';
 	}
+	if (size === '3xsmall') {
+		return '3xs';
+	}
 
 	return size;
 };
 
-const getUtilityClass = (
-	utility,
-	screens,
-	typoType,
-	size,
-	missingMediaQuery
-) => {
+const getUtilityClass = (utility, scale, textType, size) => {
+	const isHeadline = textType === 'headline';
+
 	let result = `
-${utility ? '.' : '%'}${prefix}-typo-${typoType}-${getShortSize(size)}{
+${utility ? '.' : '%'}${prefix}-${scale}-${textType}-${getShortSize(size)}{
 `;
-	if (!missingMediaQuery) {
-		result += `
-\tline-height: $${prefix}-typography-${typoType}-${size}-mobile-line-height;
-\tfont-size: $${prefix}-typography-${typoType}-${size}-mobile-font-size;
-\tfont-weight: $${prefix}-typography-${typoType}-${size}-mobile-font-weight;
+	result += `
+\tline-height: $${prefix}-typography-${scale}-mobile-${textType}-${size}-line-height;
+\tfont-size: $${prefix}-typography-${scale}-mobile-${textType}-${size}-font-size;
 `;
 
+	if (isHeadline) {
 		result += `
-\t@media only screen and (min-width: ${screens.md.value}) {
-\t\tline-height: $${prefix}-typography-${typoType}-${size}-tablet-line-height;
-\t\tfont-size: $${prefix}-typography-${typoType}-${size}-tablet-font-size;
-\t\tfont-weight: $${prefix}-typography-${typoType}-${size}-tablet-font-weight;
+    &-light,
+    &[data-variant="light"] {
+        font-weight: 300;
+    }
```

Comment:
Could headlines really include light elements within as well?

Reply:
As we discussed, we need it because we have two font styles as we have for regular body tags. The difference is that we have a HTML tag (<strong> or <b>) to change the font, but we don't have it for headlines.